src/Service/CroppedService.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Picture;
+use App\Repository\PictureRepository;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class CroppedService
+{
+    private Environment $twig;
+    private PictureRepository $pictureRepository;
+    private ParameterBagInterface $parameterBag;
+    public function __construct(
+        Environment $twig,
+        PictureRepository $pictureRepository,
+        ParameterBagInterface $parameterBag
+    ) {
+        $this->twig = $twig;
+        $this->pictureRepository = $pictureRepository;
+        $this->parameterBag = $parameterBag;
+    }
+    public function cropped(Picture $picture): Response
+    {
+        $html =  $this->twig->render('picture/cropped.html.twig', [
+            'picture' => $picture,
+        ]);
+        return new Response($html);
```

Comment:
The "return new Response(...)" is the responsibility of the controller, it should not be in a service.